examples/newrelic/newrelic-daemonset.yaml

```diff
@@ -0,0 +1,58 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  labels:
+    name: newrelic
```

Comment:
I would suggest something like:
```yaml
labels:
  tier: monitoring
  app: newrelic-agent
  version: v1
```
This 10 of [config best pracitces](https://github.com/kubernetes/kubernetes/blob/master/docs/user-guide/config-best-practices.md)